install/install.sh

```diff
@@ -38,6 +38,18 @@ usage() {
          -e server_port=8443
     * Specify server container name (defaults to quipucords):
          -e server_name=qpc_server
+    * Specify postgresql as database management system (default is to use SQLite):
```

Comment:
Remove all the changes to this file as @abaiken covers this in her PR (see https://github.com/quipucords/quipucords/pull/1439)